### PR TITLE
feat: add salespages link to home splash screen

### PR DESCRIPTION
**Pull request by raphael-arce**
- add salespages banner to home splash screen
- fixed typos and localized content for the banner
- move matomo tracking function to utils

### CODE REVIEW

**Review by ff6347 — COMMENTED** [review]
LGTM but why the merge into the feat/move-matomo-to-script-tag and not into staging?

**Review by ff6347 — CHANGES_REQUESTED** [review]
image

Looks pretty good. The last FAQ block is still a little off.

**Comment by raphael-arce** [comment]
> LGTM but why the merge into the feat/move-matomo-to-script-tag and not into staging?

thanks! when I created the PR, feat/move-matomo-to-script-tag was not merged into staging yet, and I thought it would probably make sense once to point it towards that branch as long it's not merged (since it's based on code that was added in that branch). Does that make any sense?

**Comment by ff6347** [comment]
> > LGTM but why the merge into the feat/move-matomo-to-script-tag and not into staging?
> 
> thanks! when I created the PR, feat/move-matomo-to-script-tag was not merged into staging yet, and I thought it would probably make sense once to point it towards that branch as long it's not merged (since it's based on code that was added in that branch). Does that make any sense?

Yes. Stacked PRs. Makes sense.

**Comment by raphael-arce** [comment]
> image
> 
> Looks pretty good. The last FAQ block is still a little off.

Good catch 👍  I Created a Ticket to investigate & fix it: https://app.asana.com/0/1204152181784036/1206725176123521/f

File: src/components/Credits/index.tsx [hunk 1]
```diff
@@ -2,8 +2,7 @@ import React, { FC } from 'react';
 import styled from 'styled-components';
 import useLocalizedContent from '../../utils/hooks/useLocalizedContent';
 
-const logoCitylab =
-  'https://logos.citylab-berlin.org/logo-citylab-berlin-outline.svg';
+const logoCitylab = 'https://logos.citylab-berlin.org/logo-citylab-color.svg';
```
Comment (ff6347) [inline src/components/Credits/index.tsx:5]: Why are we linking to external sites for logos?
Also shouldn't logos not be part if content?

**Review by ff6347 — APPROVED** [review]
Looks good. Except for the logos I'm fine with that. But let's take the logos on external and as content into the next sprint as ticket

File: src/components/Credits/index.tsx (hunk 1 above)
Comment (raphael-arce) [reply src/components/Credits/index.tsx:5]: valid point. I guess in the past the idea was to link it from a single source of truth so updating logos becomes easier. But I agree we should discuss how to handle this (as for example, an accidental change in the source wrongly changed the logos on many websites).

**Pull request merged**